coverage/annotate.py

```diff
@@ -60,6 +60,7 @@ def annotate_file(self, fr, analysis):
         statements = sorted(analysis.statements)
         missing = sorted(analysis.missing)
         excluded = sorted(analysis.excluded)
+        missing_branches = sorted(analysis.missing_branch_arcs().keys())
```

Comment:
There's no reason to sort this, since we're just using `in` on it.  Making it a set would be better.  Same for `excluded`, though that was my fault! :)   `statements` and `missing` need to be sorted because of how the algorithm walks them in tandem.